Source/Shaders/PostProcessFilters/FXAA.glsl

```diff
@@ -1,21 +1,21 @@
 varying vec2 v_textureCoordinates;
 
-uniform sampler2D u_texture;
-uniform vec2 u_fxaaQualityRcpFrame;
+uniform sampler2D u_colorTexture;
 
 const float fxaaQualitySubpix = 0.5;
 const float fxaaQualityEdgeThreshold = 0.125;
 const float fxaaQualityEdgeThresholdMin = 0.0833;
 
 void main()
 {
+    vec2 fxaaQualityRcpFrame = vec2(1.0) / czm_viewport.zw;
```

Comment:
@bagnell you might have to double check if this is correct. I removed the uniform value and just calculate this in the shader. This may be related to the multiple-viewport changes in https://github.com/AnalyticalGraphicsInc/cesium/pull/5225

Reply:
Bump - in case this is important.

Reply:
This is fine for now. This would need to change for multiple viewports, but a lot more work would need to be done to get post-processing per-viewport.